### ISSUE TITLE
[1.12.2] Morphing tool doesn't work with Logistics Pipes

### ISSUE THREAD

**Issue by lucasqueiroz** [Vazkii/Morph-o-Tool#60]
Hey there, I crafted a Morphing tool and added the BuildCraft Wrench to it, but when I aim at a Logistics Pipes' pipe, it doesn't morph to the correct tool.
Is it planned to be integrated with Logistics Pipes? Or maybe I'm using it wrong (I also tried with the IC2 wrench and it works correctly)?
Thanks

**Comment by MoreThanHidden** [comment]
In the config you'll need to add a Mod Alias "logisticspipes=buildcraft".
Otherwise can Sneak (Shift) + Scroll to the tool.

**Comment by lucasqueiroz** [comment]
I was reading the code and found it had a configuration, should've looked there before.

But I tried adding `logisticspipes=buildcraft` and it didn't work.
I also noticed that the Buildcraft pipes aren't showing the correct wrench as well, even tough the default configuration includes it, and also tried with `buildcraftall=buildcraft`, since I'm using the single JAR file (Although I don't think it should matter).
But for now the Sneak + Shift works.
Thanks!
